config.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+class day(Enum):
+    sunday = 6
+    monday = 0
+    tuesday = 1
+    wednesday = 2
+    tursday = 3
+    friday = 4
+    saturday = 5
+
+class MessageType(Enum):
+    reminder = 0
+    NewPeriod = 1
+
+
+Hour = 60*60
+_8AM = 8
```

Comment:
Also, don't see a point in these constants.

Reply:
I have been taught that leaving unattended numbers around your code is not a good practice and I can understand why.
I might agree if your tell me that these are just simple enough to understand even without the variable name...